org.gtk.Gtk3theme.Sierra-Negra.json

```diff
@@ -0,0 +1,43 @@
+{
+   "id":"org.gtk.Gtk3theme.Sierra-Negra",
+   "branch":"3.22",
+   "runtime":"org.gnome.Sdk",
+   "build-extension":true,
+   "sdk":"org.gnome.Sdk",
+   "runtime-version":"3.26",
+   "appstream-compose":false,
+   "separate-locales":false,
+   "modules":[
+      {
+         "name":"Sierra-Negra",
+         "buildsystem":"simple",
+         "build-commands":[
```

Comment:
Strange formatting, put spaces between the value.

Reply:
Oh were you saying to put a space between Sierra and Negra rather than a hyphen?